doc/manual/migrating.rst

```diff
@@ -0,0 +1,45 @@
+.. _migrating
+
+Migrating from EEGLAB
+=====================
+
+To read in data exported from EEGLAB, MNE offers an EDF reader :func:`mne.io.read_raw_edf` and a `set` file reader.
+To read in `set` files containing ``raw`` data, use :func:`mne.io.read_raw_eeglab` and to read in `set` files containing
+``epochs`` data, use :func:`mne.read_epochs_eeglab`.
+
+Here is a cheatsheet to help users migrate painlessly from EEGLAB. For the sake of clarity, let us assume
+that the following are already defined or known: the file name ``fname``, low-pass cut-off frequency ``lfreq``,
+time interval of the epochs ``tmin`` and ``tmax``, and the conditions ``cond1`` and ``cond2``.
```

Comment:
Make it explicit?

```
from mne import io
lfreq, hfreq = 1, 30

fname = "/.../"
cond1, cond2 = ...
```

And we don't necessarily need tmin and tmax anymore, with the most recent changes, don't we?